pkg/operator/dashboards.go

```diff
@@ -0,0 +1,165 @@
+package operator
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/monitoring/pkg/common"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+	"github.com/wushilin/stream"
+)
+
+const (
+	datasource_name = "tidb-cluster"
+)
+
+var (
+	overviewExlcudeItems = []string{"Services Port Status", "System Info"}
+	tikvExcludeItems     = []string{"IO utilization"}
+
+	dashboards = map[string]string{
```

Comment:
This should be configurable.

Reply:
The shell also depend on the name, so even though we support it as configurable, but we also have to need to update the shell script for now.
We will treat it in the future.